statusboard/src/components/Map/Map.js

```diff
@@ -25,14 +27,32 @@ export default function Map({ brigadeData, filterOpts, setFilterOpts }) {
       const { latitude, longitude } = filterOpts.selectedBrigade || {};
       setCenter([latitude, longitude]);
     } else {
-      setZoom(defaultZoom);
-      setCenter(defaultCenter);
+      let userCenter = [];
+      const foundLocation = (position) => {
+        userCenter = [position.coords.latitude, position.coords.longitude];
+        setZoom(2);
+        setCenter(userCenter);
+      };
+      const noLocation = () => {
+        setZoom(defaultZoom);
+        setCenter(defaultCenter);
+      };
+
+      const options = { timeOut: 20000, maximumAge: 60 * 60 * 24 };
```

Comment:
[Add 1000 to `maximumAge`](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/getCurrentPosition#:~:text=maximum%20age%20in%20milliseconds)

Reply:
Yes 🙃

Reply:
(Fixed)